ajax/libs/jquery-localize/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "jquery-localize",
+  "filename": "jquery.localize.min.js",
+  "description": "A jQuery plugin that makes it easy to i18n your static web site.",
+  "author": "coderifous",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/coderifous/jquery-localize.git"
+  },
+  "keywords": [],
```

Comment:
Please add more keywords, such as `jquery`, `localize`, `internationalize`, `i18n`.
This will good for search and help people fine this lib quickly.
Thanks.